tests/testthat/test-mgaussian.R

```diff
@@ -39,8 +48,10 @@ test_that("standardizing responses works", {
   y_standardized <- scale(y, scale = apply(y, 2, sd2))
 
   sfit <- sgdnet(x, y, family = "mgaussian", standardize.response = TRUE)
+  bfit <- sgdnet(x, y, family = "mgaussian", standardize.response = TRUE, batchsize = 10)
   gfit <- glmnet(x, y, family = "mgaussian", standardize.response = TRUE)
 
   expect_equal(sfit$lambda, gfit$lambda)
   expect_equivalent(coef(sfit), coef(gfit))
+  expect_equivalent(coef(bfit), coef(gfit), tolerance = 1e-2)
```

Comment:
This is still really loose and seems prone to false negatives: can we tighten this test?

Reply:
I have tightened the test to have smaller tolerance.